src/binder/inc/clrprivbinderassemblyloadcontext.h

```diff
@@ -80,6 +120,10 @@ class CLRPrivBinderAssemblyLoadContext : public IUnknownCommon<ICLRPrivBinder>
     CLRPrivBinderCoreCLR *m_pTPABinder;
     
     INT_PTR m_ptrManagedAssemblyLoadContext;
+
```

Comment:
hm actually maybe no, the field is used indirectly through GetManagedAssemblyLoadContext() which was already there before collectible ALC.